### PR TITLE
Travel Fund Request for @bamieh at Collab Summit Berlin 2019

### PR DESCRIPTION
**Pull request by Bamieh**
Event: Node.js Collaborator's Summit
When: 30 May - 31 May 2019
Event Location: Berlin, Germany
Traveling From: Amman Jordan
Stipend Size: 1200 USD

Breakdown:
- Airplane: $700
- Hotel: $300 (2 nights)
- Visa: $150
- Misc: $50

I am planning to attend. Hopefully if I get the Visa in time I will submit a session proposal to speak about mentorship.

### CODE REVIEW

File: TravelFunds/2019.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ Guy        |Bedford        |Collaborator Summit   |Attendance | | Berlin
...
 Ruben      |Bridgewater    |Collaborator Summit   |Attendance | | Berlin         |30 May - 31 May 2019 |650 €                 | 28 March 2019          |  https://github.com/nodejs/admin/pull/322    |                               |                               |    |                                   |                                                     |                               |
 Mohit        |Bajoria       |Collaborator Summit   |Attendance | | Berlin         |30 May - 31 May 2019 |1500 USD                 | 28 March 2019          |                                         |                               |                               |    |                                   |                                                     |                               |
+Ahmad        |Bamieh       |Collaborator Summit   |Attendance | | Berlin         |30 May - 31 May 2019 |1200 USD                 | 10 April 2019          |                                         |                               |                               |    |                                   |                                                     |                               |
```
Comment (WaleedAshraf) [inline TravelFunds/2019.md]: You can also add
PR: https://github.com/nodejs/admin/pull/335

Comment (Bamieh) [reply TravelFunds/2019.md]: @WaleedAshraf I'm not sure I follow?

**Comment by keywordnew** [comment]
Your open PR mentioned applying for a visa. Just a reminder that visa letters are available this year!

File: TravelFunds/2019.md (hunk 1 above)
Comment (WaleedAshraf) [reply TravelFunds/2019.md]: There is a column for adding PR link, you can add it.
I think it can be done later though.

Comment (Bamieh) [reply TravelFunds/2019.md]: Ah got it. Thanks for pointing it out.

**Comment by Bamieh** [comment]
@keywordnew yes indeed! i've already got my invitation for the visa!

**Pull request merged**